data/aliases.json

```diff
@@ -0,0 +1,8 @@
+[
+  ["\\mathfrak", "\\frak"],
+  ["\\mathcal", "\\cal"],
+  ["\\mathbb", "\\bb"],
+  ["\\mathbf", "\\bf"],
+  ["\\dfrac", "\\frac"],
+  ["\\ldots", "\\dots"]
+]
```

Comment:
I have to admit that I do not know much about LaTeX. Actually, I just translated the Ruby code from @vikhyat, as you see, those are the aliases he used [here](https://github.com/vikhyat/latex-to-unicode/blob/master/lib/data/aliases). 

I just made some researches but I was not able to find much info about aliases in LaTeX. However, I found [a question on StackOverflow](http://stackoverflow.com/questions/28444744/how-do-i-alias-command-mathbb), and an user is asking how to declare `\\bb` as the alias of `\\mathbb`. So, what I understood is that there is not any "built-in" alias in LaTeX, am I wrong? So this file may be used for users whose want to have custom aliases.